hatch/app.py

```diff
@@ -20,8 +20,7 @@ def validate(auth_token: str = Security(api_key_header)):
 
 @app.get(
     "/workspace/{workspace}",
-    response_model=FilesIndex,
-    dependencies=[Depends(validate)],
```

Comment:
Why doesn't this depend on `validate` any more?

Reply:
Because the whole FastAPI app Depends on it - this view specific Depends was left over from previous iterations, I just noticed it and cleaned it up in this PR. It effectively did the validation twice for this url

Reply:
Ah, I missed the whole app depending on it, nice!